tests/test_qnode.py

```diff
@@ -559,7 +564,7 @@ def circuit(w):
         assert np.allclose(circuit_output, expected_output, atol=tol, rtol=0)
 
         # circuit jacobians
-        circuit_jacobian = circuit.jacobian(multidim_array)
+        circuit_jacobian = circuit.jacobian([multidim_array])
```

Comment:
what caused this change?

Reply:
`QNode.jacobian` accepts a list of parameters, so the previous function call was incorrect. Not sure why it passed the tests with caching turned on, but it failed once I turned caching off.